WP_Auth0.php

```diff
@@ -167,7 +167,7 @@ function wp_auth0_register_widget() {
 add_action( 'widgets_init', 'wp_auth0_register_widget' );
 
 function wp_auth0_register_query_vars( $qvars ) {
-	return array_merge( $qvars, [ 'error', 'error_description', 'a0_action', 'auth0', 'state', 'code' ] );
+	return array_merge( $qvars, [ 'error', 'error_description', 'a0_action', 'auth0', 'state', 'code', 'invitation', 'organization', 'organization_name' ] );
```

Comment:
It's passed along with ?invitation and ?organization as part of the invitation flow. Not currently using it in the core logic here, but as WordPress is kind of a LEGO set for building custom solutions on top of (see examples directory), I'm including it here in case customers might want to work with the variable at some point.

Reply:
Thanks !